app/models/roles_metadata_banned_competitors.rb

```diff
@@ -1,4 +1,8 @@
 # frozen_string_literal: true
 
 class RolesMetadataBannedCompetitors < ApplicationRecord
+  enum :scope, {
+    competing_only: "competing_only",
+    competing_and_attending: "competing_and_attending",
```

Comment:
I'm assuming you discussed this with WDC, but what about other scopes like `organizing` or `delegating`?

Reply:
As of now WDC asked for only these two scopes and they might increase them in future.